game/templates/game/login.html

```
@@ -29,50 +31,58 @@
+                        {% if field.errors %}
+                            {{ field.errors }}
+                        {% endif %}
                     </div>
-                    {% if field.errors %}
-                        {{ field.errors }}
-                    {% endif %}
+                {% endfor %}
+
+                <div class="remember-me modern-remember">
+                    <label class="custom-checkbox-container">
+                        <input type="checkbox" name="remember_me">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)